recbole/model/sequential_recommender/fossil.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
+# @Time    : 2020/11/21 20:00
+# @Author  : Shao Weiqi
+# @Reviewer : Lin Kun
+# @Email   : shaoweiqi@ruc.edu.cn
+
+r"""
+FOSSIL
+################################################
+
+Reference:
+    Ruining He et al. "Fusing Similarity Models with Markov Chains for Sparse Sequential Recommendation." in IEEE 2016
```

Comment:
Ruining He et al. "Fusing Similarity Models with Markov Chains for Sparse Sequential Recommendation." in ICDM 2016.

Reply:
resolved